File: scripts/restore_lightweight_models.py
import os
import shutil

src_dir = r'c:\Users\digit\LS Furniture APP\models_deploy_temp'
dst_dir = r'c:\Users\digit\LS Furniture APP\ls-lifestyle-web\public\assets\models'

if not os.path.exists(dst_dir):
    os.makedirs(dst_dir)

files = os.listdir(src_dir)
moved_count = 0

for f in files:
    src_path = os.path.join(src_dir, f)
    if os.path.isfile(src_path):
        size_mb = os.path.getsize(src_path) / (1024 * 1024)
        if size_mb < 2.0:
            dst_path = os.path.join(dst_dir, f)
            print(f"Moving {f} ({size_mb:.2f} MB)")
            shutil.move(src_path, dst_path)
            moved_count += 1

print(f"Successfully moved {moved_count} lightweight models.")

File: scripts/restore_lightweight_models.js
const fs = require('fs');
const path = require('path');

const srcDir = 'c:\\Users\\digit\\LS Furniture APP\\models_deploy_temp';
const dstDir = 'c:\\Users\\digit\\LS Furniture APP\\ls-lifestyle-web\\public\\assets\\models';

if (!fs.existsSync(dstDir)) {
    fs.mkdirSync(dstDir, { recursive: true });
}

const files = fs.readdirSync(srcDir);
let movedCount = 0;

files.forEach(f => {
    const srcPath = path.join(srcDir, f);
    if (fs.lstatSync(srcPath).isFile()) {
        const stats = fs.statSync(srcPath);
        const sizeMb = stats.size / (1024 * 1024);
        if (sizeMb < 2.0) {
            const dstPath = path.join(dstDir, f);
            console.log(`Moving ${f} (${sizeMb.toFixed(2)} MB)`);
            fs.renameSync(srcPath, dstPath);
            movedCount++;
        }
    }
});

console.log(`Successfully moved ${movedCount} lightweight models.`);
